pyaerocom/io/read_eea_aqerep_base.py

```diff
@@ -278,8 +294,10 @@ def read_file(self, filename, var_name,
             lineidx = 0
             for line in f:
                 rows = line.rstrip().split(file_delimiter)
+                # skip line if the # rows is not sufficient
+                if len(rows) < max_file_index_to_keep:
```

Comment:
I guess this is the fix for #404 / #405. Looks good.

Reply:
I just found another error while testing with all zipped source files. updated PR coming soon...

Reply:
forgot to increase version number of the reader

Reply:
merging PR...